qa/runtests.sh

```diff
@@ -7,7 +7,7 @@ pushd "${SCRIPT_DIR}/.." > /dev/null
 set -e
 set -x
 
-COVERAGE_THRESHOLD=90
+COVERAGE_THRESHOLD=80
```

Comment:
it's not a great idea to change the threshold

Reply:
changed it back to 90